### ISSUE TITLE
EX - Remove "Austin Transportation (ATD)" (50694) and "Public Works Department PWD" (50690) and Add new Transportation and Public Works TPW (LP)  (On PROD)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#12141]
** Acceptance Criteria ** 

- The subtypes Austin Transportation (ATD) (50694) and Public Works Department PWD (50690) need to be removed from Back Office and Portal on EX Folders. 
- A new subtype needs to be added for the EX folders named 'Transportation and Public Works (TPW)'.
- The new subtype needs to have info fields that were available in both ATD and PWD subtypes. 
- The new subtype needs to insert and run fees that were available in both ATD and PWD subtypes. 
- The new subtype needs to have the same process workflow as the ATD and PWD folders (stakeholders mentioned there aren't any differences in folder process workflow).
- The new subtype needs to be made available on the ABC Portal. Users should be able to apply for this subtype when creating an EX application. 
- Any historical EX records with ATD and PWD subtypes need to be retained and viewable in the Back Office and ABC Portal.  

> What application are you using?

AMANDA

> Describe the problem.

The EX folder Sub Types "Austin Transportation (ATD)" (50694) and "Public Works Department PWD" (50690) should be merged into a single Sub Type for the new Transportation and Public Works Department (TPW).

> Describe the outcome you'd like to see when this feature is implemented.

The Sub Types should reflect the current department TPW, which formed as a merger of ATD and PWD so we can represent the new department name more accurately and keep customers aware of the current nomenclature.

> Describe any workarounds you currently have in place or alternative solutions you've considered.

N/A

> Requested By
Kyle K.

Request ID: DTS23-107415

**Comment by TracyLinder** [comment]
@vschandramouli please prioritize scoping this issue. If we can have some feedback by next Friday 7/14 that would be great.

**Comment by vschandramouli** [comment]
Updated AC, moving to Backlog.

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Committed changes and requested for Dev/Test promotion
SCTASK0351771

**Comment by Nadin-Nader** [comment]
Ready for UAT on TEST

**Comment by Nadin-Nader** [comment]
Priya is debugging

**Comment by Nadin-Nader** [comment]
Sent Kyle a message via Teams asking for UAT on TEST. Waiting on response

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been approved by Kyle to be deployed to PROD

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Created Prod deployment request - SCTASK0361104

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT This has been deployed to PROD

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT i asked Kyle to UAT this on PROD. Waiting on Response

**Comment by Nadin-Nader** [comment]
Per Kyle's UAT this is good to go on PROD. closing
